homeassistant/components/litterrobot/entity.py

```diff
@@ -146,3 +150,20 @@ async def perform_action_and_assume_state(
         if await self.perform_action_and_refresh(action, assumed_state):
             self._assumed_state = assumed_state
             self.async_write_ha_state()
+
+
+def async_update_unique_id(
+    hass: HomeAssistant, domain: str, entities: Iterable[LitterRobotEntity]
+) -> None:
+    """Update unique ID to be based on entity description key instead of name.
+
+    Introduced with release 2022.9.
```

Comment:
@MartinHjelmare is this worthy of beta? Or should the release here be adjusted?

Reply:
Also, what's the standard on how long code like this should be left in place before being removed in a future version?

Reply:
I think we can add it to the beta.

Reply:
I'd say between two releases and half a year.